CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change log
 
+- [#574](https://github.com/mobilityhouse/ocpp/issues/574) Remove v1.6 deprecated enums
```

Comment:
Maybe we can collect backwards incompatible changes like these in a "Breaking changes" section.
Each entry should include a link to an issue which explains the breaking change and suggests an upgrade path. I added a description to https://github.com/mobilityhouse/ocpp/issues/574 that you can link to.

And a nitpick about the description itself. No enums have been removed. Only enum variants.

Reply:
So to confirm given #574, would this description be a complete example? Meaning it can be used as a template for other breaking changes into the future?

Reply:
This should now be done